script/Ciel/Day 5/05_01_CIEL05_2_2.txt

```diff
@@ -410,47 +436,52 @@ Even I can see that's she's trying to stir him up.
 [sha:703956460ea11c6e104f855770882c0f3ca5e9a8]{
 -- Page 331, Offset 11747.
 -- 「“……その通りだ、代行者。
-"...Indeed, Executor.
+"It is as you say, Executor...
```

Comment:
Petrikow was saying that for these pauses, we should add ellipses. can't check ig right now though